draft-ietf-quic-transport.md

```diff
@@ -3473,6 +3473,7 @@ to be greater than the number of packets acknowledged in an ACK frame.  When
 this happens, and if validation succeeds, the local reference counts MUST be
 increased to match the counts in the ACK frame.
 
```

Comment:
Related note: I just realised that we don't test the case where ALL ECT get remarked to CE which should also lead to failure.